solver/src/settlement_submission/flashbots_api.rs

```diff
@@ -82,7 +82,10 @@ impl FlashbotsApi {
                 },
                 jsonrpc_core::Output::Failure(body) => Err(anyhow!(body.error)),
             },
-            Err(err) => Err(anyhow!(err)),
+            Err(err) => {
+                tracing::error!("flashbot cancellation response: {}", body);
```

Comment:
Nit: every error severity log will lead to an alert in our slack channel. Given we only want this for debugging maybe a warning or info severity is sufficient?

Reply:
Changed to info.